package.json

```diff
@@ -89,6 +89,8 @@
     "publish:all": "npm publish dist/npm/clarity-ui; npm publish dist/npm/clarity-angular; npm publish dist/npm/clarity-icons",
     "presample-app": "npm i dist/npm/clarity-ui; npm i dist/npm/clarity-icons; npm i dist/npm/clarity-angular;",
     "sample-app": "gulp sample",
+    "preks:publish": "gulp npm:prepare --version $npm_package_version",
+    "ks:publish": "cd src/ks-app; ng build --preserve-symlinks --prod; cd ../../dist/ks-app; cat index.html > 404.html; surge; cd ../../",
```

Comment:
This is awesome and will help a lot. Thanks @hippee-lee!

@jeeyun can we do this for our main demo app too with the webpack change? It will make adding demos to reviews so much easier & faster.

Reply:
btw I am talking about surge.sh here :D

Reply:
Yes, shouldn't be difficult to add this.